Harvard-Stellenbosch-University.csl

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard - Stellenbosch University  - Yusuf Ras</title>
+    <id>http://csl.mendeley.com/styles/21406231/harvard-stellenbosch-university--2</id>
```

Comment:
You can see all the formal requirements [here](https://github.com/citation-style-language/styles/wiki/Style-Requirements). As per 5. this should be ```http://www.zotero.org/styles/harvard-stellenbosch-university```